addons/web/tests/test_perf_load_menu.py

```diff
@@ -38,7 +38,7 @@ def test_load_web_menus_perf(self):
         self.env.registry.clear_all_caches()
         self.env.invalidate_all()
         # cold orm/fields cache (only web: 10, all module: 40 - no demo)
-        with self.assertQueryCount(40):
+        with self.assertQueryCount(38):
```

Comment:
The two query counters here were set to 40 to make them pass without demo data (as indicated by the comment above 🤭 )

See: https://github.com/odoo/odoo/pull/190328/files